pkg/virt-operator/creation/rbac/apiserver.go

```diff
@@ -256,6 +256,22 @@ func newApiServerClusterRole() *rbacv1.ClusterRole {
 					"watch", "list",
 				},
 			},
+			{
+				APIGroups: []string{
+					"certificates.k8s.io",
+				},
+				Resources: []string{
+					"certificatesigningrequests",
+				},
+				Verbs: []string{
+					"create",
+					"update",
```

Comment:
does the apiserver need "update" and "delete" privs here for csr?

Reply:
They should not have been there at all. There is an extra clusterrole which the components are added to. Removed. Good catch.